File: sdks/ruby/lib/novabilling/subscriptions/types/create_subscription_dto_status.rb
# frozen_string_literal: true

module Novabilling
  module Subscriptions
    module Types
      module CreateSubscriptionDtoStatus
        extend Novabilling::Internal::Types::Enum

        ACTIVE = "ACTIVE"
        TRIALING = "TRIALING"
        PAUSED = "PAUSED"
        PAST_DUE = "PAST_DUE"
        CANCELED = "CANCELED"
      end
    end
  end
end

File: sdks/php/src/Subscriptions/Types/CreateSubscriptionDtoStatus.php
<?php

namespace NovaBilling\Subscriptions\Types;

enum CreateSubscriptionDtoStatus: string
{
    case Active = "ACTIVE";
    case Trialing = "TRIALING";
    case Paused = "PAUSED";
    case PastDue = "PAST_DUE";
    case Canceled = "CANCELED";
}
